apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -1,67 +1,99 @@
-<div class="container-lg px-4 lg:mx-auto">
-  <p
-    class="font-title text-[28px] font-medium mt-8 mb-5 text-title text-center sm:mt-12 sm:mb-[-22px] sm:text-left"
-    translate
+<ng-container *ngIf="apiLinks$ | async as apiLinks">
```

Comment:
All the data contened inside the template is base on apiLinks$.
So if I add this ng if at the root inside an ngContainer, I'm garanted that I have it.
And more importantly, with that technique my template makes only one subscribe as I can work with the created variable apiLinks in the rest of the template (since I need it for .length and the ngFor)

Reply:
I think reading the length of apiLinks is not needed anymore for the carousel; also having several `| async` is fine IMO, it's one of those verbose things in Angular but adding an ngIf just for the sake of writing less down the line can be a bit confusing (other people might wonder why this ngIf is there in the first place)

Reply:
FYI https://ngrx.io/guide/component/let

Reply:
thanks! I knew I had seen something like this :+1: